src/ontology/cl-edit.owl

```diff
@@ -29729,6 +29730,18 @@ AnnotationAssertion(rdfs:label obo:CL_4030006 "fallopian tube secretory epitheli
 SubClassOf(obo:CL_4030006 obo:CL_1001591)
 SubClassOf(obo:CL_4030006 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003889))
 
+# Class: obo:CL_4030007 (fallopian tube ciliated cell)
+
+AnnotationAssertion(Annotation(<http://purl.org/dc/elements/1.1/contributor> "http://orcid.org/0000-0002-0819-0473"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:16565155"^^xsd:string) obo:IAO_0000115 obo:CL_4030007 "An epithelial ciliated cell that is part of the fallopian tube. This cell type is found mainly on the apex of the mucosal folds and contributes, together with peristaltic contractions, to the self-propulsion of spermatozoa, the transport of ovum during ovulation and the transport of the fertilized ovum to the intramural fallopian tube. A fallopian tube ciliated cell has a columnar shape and contains an oval or round nucleus, often located perpendicular or parallel to the long axis of the cell."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4030007 <http://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_4030007 "2022-03-08T12:40:19Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4030007 "oviduct ciliated cell"^^xsd:string)
```

Comment:
@bvarner-ebi I'd make this a broad synonym please, as not all oviducts have Fallopian tubes (e.g. in non-mammals)

Reply:
Thank you! In that case, I also update the synonym type for the 'fallopian tube secretory epithelial cell' as well.